tests/test_modeling_common.py

```diff
@@ -484,123 +484,119 @@ def test_training_gradient_checkpointing(self):
             loss.backward()
 
     def test_attention_outputs(self):
-        if not self.has_attentions:
-            pass
+        config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
+        config.return_dict = True
 
-        else:
-            config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
+        seq_len = getattr(self.model_tester, "seq_length", None)
```

Comment:
I'm not sure why the diff is so confusing for this file (possibly blank lines?). The only change is:
* Removing the if / else statement
* Un-indenting the logic in the else block

Apologies for the difficulty this introduces when reviewing.